src/test/scala/TriggerRewriterTests.scala

```diff
@@ -32,15 +34,20 @@ class TriggerRewriterTests extends AnyFunSuite with Matchers {
     }
   }
 
-  val x0 = Var(Identifier("x0"), sorts.Int)
-  val x1 = Var(Identifier("x1"), sorts.Int)
-  val y0 = Var(Identifier("y0"), sorts.Int)
-  val z0 = Var(Identifier("z0"), sorts.Int)
+  lazy val x0 = Var(Identifier("x0"), sorts.Int)
```

Comment:
Because calling ``Var`` and ``Identifier`` now calls the apply methods, which read ``Verifier.config.useFlyweight``, which is only initialized in the ``before`` block of the test. So if I make this a normal val, i get a NPE.